### DISCUSSION TITLE
Alpine error when livewire removes an element

### DISCUSSION THREAD

**Discussion by argonzalez2016** [alpinejs/alpine#3010] [1. Help]
Hello. I am getting the following error when livewire removes an element on the page.
```
Alpine Expression Error: Cannot read properties of undefined (reading 'attribute')

Expression: "products.payable.base.attributes[4].attribute"
```

This happens after livewire removes the "attribute[4]" key after I click a button to remove the element containing the model "products.payable.base.attributes[4].attribute"

Both Livewire and Alpine (Through entangle) remove the key in the data, however alpine doesn't recognize the element is missing and still searches for that attribute, causing and error and breaking my page.

Any help would be appreciated.

**Comment by masterix21** [comment]
@argonzalez2016, I'm facing the same issue. Have you ever found a solution?
